app/ui/browser/views/browser.jsx

```diff
@@ -170,10 +170,6 @@ function attachIPCRendererListeners(browserView) {
   ipcRenderer.on('profile-diff', (_, args) => dispatch(args));
   ipcRenderer.on('focus-url-bar', () => dispatch(ipcActions.focusURLBar()));
   ipcRenderer.on('new-tab', () => dispatch(actions.createTab()));
-
-  // @TODO main process should be sending an id to close a tab
```

Comment:
Is this TODO resolved?  We are still sending a close-tab event without an ID from the main process via the Window menu